jekyll/_cci2/docker-layer-caching.md

```diff
@@ -26,13 +26,13 @@ Docker Layer Caching can be used with both the [`machine` executor]({{ site.base
 
 **Note:** DLC has **no** effect on Docker images used as build containers. That is, containers that are used to _run_ your jobs are specified with the `image` key when using the [`docker` executor]({{ site.baseurl }}/2.0/executor-types/#using-docker) and appear in the Spin up Environment step on your Jobs pages.
 
-## Enabling DLC
```

Comment:
Agreed, and this section heading should stay "Enabling DLC" where we talk about support

Reply:
rest looks good

Reply:
that header is for not just the message about contacting support but also the subsequent two sub-sections about the RDE & machine exec, that's y the first 1 is a ## header & the next two are ### headers..... i'm fine w/ it reading 'enabling DLC' tho so i will make the change